components/nexus-security/src/main/java/org/sonatype/nexus/security/WebSecurityModule.java

```diff
@@ -56,7 +64,7 @@
  *
  * @since 2.6.1
  *
- * @see SecurityModule
+ * @see TestSecurityModule
```

Comment:
I left it here until TestSecurityModule was removed, which will be soon... though its still used no?  I can remove the @see ref now though.